core/src/main/java/com/sequenceiq/cloudbreak/core/bootstrap/service/host/ClusterHostServiceRunner.java

```diff
@@ -656,12 +659,22 @@ public Map<String, SaltPillarProperties> createPillarWithClouderaManagerSettings
                                 "missed_heartbeat_interval", cmMissedHeartbeatInterval,
                                 "disable_auto_bundle_collection", disableAutoBundleCollection,
                                 "set_cdp_env", isVersionNewerOrEqualThanLimited(cmVersion, CLOUDERAMANAGER_VERSION_7_0_2),
+                                "cloud_provider_setup_supported", isCloudProviderSetupSupported(stackDto, cmVersion),
+                                "cloud_provider", stackDto.getCloudPlatform(),
```

Comment:
Cloud provider (e.g. the platform) is already available at the top level pillar, with `salt['pillar.get']('platform')`